app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,5 @@
 class WelcomeController < ApplicationController
+  layout 'nonavbar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.